recipe/meta.yaml

```diff
@@ -1,28 +1,27 @@
 {% set name = "nose2" %}
 {% set version = "0.10.0" %}
```

Comment:
Should we update to the latest of `allure-python` requires a specific version?

Reply:
updated to `0.13.0`, thanks @JeanChristopheMorinPerso